setup.py

```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```

Comment:
@palewire 
On django 1.7.10 some tests fail.

**new_obj.yaml** is not converted to **python obj** but remains **unicode**. Therefore the following tests fail:

- test_blank_yaml_field
- test_nested_ordereddict
- test_ordereddict
- test_yaml_field_create
- test_yaml_field_load
- test_yaml_list

Once we have this fixed, we can change the condition mentioned by @funkybob  to include 1.7 as well. 

On django 1.8.x and django 1.9rc1 everything works like a charm.